src/org/omegat/gui/editor/MarkerController.java

```diff
@@ -88,6 +89,7 @@ public static void init() throws Exception {
         Core.registerMarker(new ComesFromMTMarker());
         Core.registerMarker(new FontFallbackMarker());
         Core.registerMarker(new SpellCheckerMarker());
+        Core.registerMarker(new AltTranslationsMarker());
```

Comment:
```suggestion
```

When registering as plugin, the marker object will be initialized in `MarkerController` constructor see around line 100